tests/test_referenceObjectLoader.py

```diff
@@ -481,6 +534,97 @@ def test_requireProperMotion(self):
             )
 
 
+class Version1ReferenceObjectLoaderLoadTests(ReferenceObjectLoaderLoadTests):
+    """This runs the same `test_properMotion` as the above class, but for a
+    version 1 reference catalog which has difference parameter names. There
+    is also no coordinate covariance, which was not available in version 1.
+    """
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        refSchema = ReferenceObjectLoader.makeMinimalSchema(filterNameList=["a"])
+        refSchema.addField("pm_ra", type="Angle")
+        refSchema.addField("pm_dec", type="Angle")
+        refSchema.addField("pm_raErr", type=np.float64)
+        refSchema.addField("pm_decErr", type=np.float64)
+        refSchema.addField("epoch", type=np.float64)
+        refCat = afwTable.SimpleCatalog(refSchema)
+        ref_md = PropertyList()
+        ref_md.set("REFCAT_FORMAT_VERSION", 1)
+        refCat.table.setMetadata(ref_md)
+        record = refCat.addNew()
+        record['coord_ra'] = lsst.geom.Angle(0)
+        record['coord_dec'] = lsst.geom.Angle(0)
+        record['coord_raErr'] = 1e-8
+        record['coord_decErr'] = 1e-8
+        pm_dir_rad = cls.properMotionDir.asRadians()
+        record['pm_ra'] = cls.properMotionAmt*np.cos(pm_dir_rad)
+        record['pm_dec'] = cls.properMotionAmt*np.sin(pm_dir_rad)
+        record['pm_raErr'] = 1e-8
+        record['pm_decErr'] = 1e-8
+        record['epoch'] = cls.epoch.mjd
+        deferredRefCat = InMemoryDatasetHandle(refCat, storageClass="SourceCatalog")
+
+        corners = [lsst.geom.SpherePoint(-1, -1, lsst.geom.degrees).getVector(),
+                   lsst.geom.SpherePoint(1, -1, lsst.geom.degrees).getVector(),
+                   lsst.geom.SpherePoint(1, 1, lsst.geom.degrees).getVector(),
+                   lsst.geom.SpherePoint(-1, 1, lsst.geom.degrees).getVector()]
+        testDataId = Struct(region=ConvexPolygon(corners))
+        cls.loader = ReferenceObjectLoader([testDataId],
+                                           [deferredRefCat],
+                                           name="testrefcat")
+
+    def test_properMotion(self):
+        """Test proper motion correction."""
+        """loaderConfig = ReferenceObjectLoader.ConfigClass()
+        loaderConfig.filterMap = {'aprime': 'a'}
+        loader = ReferenceObjectLoader([dataRef.dataId for dataRef in self.datasetRefs],
+                                       self.handles,
+                                       name="testrefcat",
+                                       config=loaderConfig)"""
```

Comment:
I see why you had to override `setUp` (to make a v1-style refcat), but why did you have to override this method? It should "just work" as-is, I'd think.

Also, you have a commented-out block here.

Reply:
Thanks for catching the commented-out block. I had actually decided I didn't want to inherit from `ReferenceObjectLoaderLoadTests`, because the other tests don't need to be rerun for version one, but then I didn't change the class definition.